src/foundry/client-esm/nue/tour.d.mts

```diff
@@ -1,9 +1,276 @@
+   */
+  get status(): Tour.STATUS;
+
+  /**
+   * Advance the tour to a completed state.
+   */
+  complete(): Promise<void>;
+
+  /**
+   * Exit the tour at the current step.
+   */
+  exit(): void;
+
+  /**
+   * Reset the Tour to an un-started state.
+   */
+  reset(): Promise<void>;
+
+  /**
+   * Start the Tour at its current step, or at the beginning if the tour has not yet started.
+   */
+  start(): Promise<void>;
+
+  /**
+   * Progress the Tour to the next step.
+   */
+  next(): Promise<void>;
+
+  /**
+   * Reward the Tour to the previous step.
+   */
+  previous(): Promise<void>;
+
+  /**
+   * Progresses to a given Step
+   * @param stepIndex - The step to progress to
+   */
+  progress(stepIndex: number): Promise<void>;
+
+  /**
+   * Query the DOM for the target element using the provided selector
+   * @param selector - A CSS selector
+   * @returns The target element, or null if not found
+   */
+  protected _getTargetElement(selector: string): Element | null;
+
+  /**
+   * Creates and returns a Tour by loading a JSON file
+   * @param filepath The path to the JSON file
+   */
+  static fromJSON(filepath: string): Promise<Tour>;
+
+  /**
+   * Set-up operations performed before a step is shown.
+   * @virtual
+   */
+  protected _preStep(): Promise<void>;
+
+  /**
+   * Clean-up operations performed after a step is completed.
+   * @virtual
+   */
+  protected _postStep(): Promise<void>;
+
+  /**
+   * Renders the current Step of the Tour
+   */
+  protected _renderStep(): Promise<void>;
+
+  /**
+   * Reloads the Tour's current step from the saved progress
+   * @internal
+   */
+  _reloadProgress(): void;
+}
+
+declare namespace Tour {
+  /** Tour configuration data */
+  interface Config {
+    /** The namespace this Tour belongs to. Typically, the name of the package which implements the tour should be used */
+    namespace: string;
+
+    /** A machine-friendly id of the Tour, must be unique within the provided namespace */
+    id: string;
+
+    /** A human-readable name for this Tour. Localized. */
+    title: string;
+
+    /** The list of Tour Steps */
+    steps: Tour.Step[];
+
+    /** A human-readable description of this Tour. Localized. */
+    description?: string;
+
+    /** A map of localizations for the Tour that should be merged into the default localizations */
+    localization?: Localization.Translations; // TODO: Verify
+
+    /** Whether the Tour is restricted to the GM only. Defaults to false. */
+    restricted?: boolean;
+
+    /** Whether the Tour should be displayed in the Manage Tours UI. Defaults to false. */
+    display?: boolean;
+
+    /** Whether the Tour can be resumed or if it always needs to start from the beginning. Defaults to false. */
+    canBeResumed?: boolean;
+
+    /** A list of namespaced Tours that might be suggested to the user when this Tour is completed. The first non-completed Tour in the array will be recommended. */
+    suggestedNextTours?: string[];
```

Comment:
Could you add `| undefined` to each of these properties? I explained why this matters on Discord but if you want I can explain again, it was in a wall of text.